docker/etc/OpenCL/vendors/nvidia.icd

```diff
@@ -0,0 +1 @@
+libnvidia-opencl.so.1
```

Comment:
I am assuming this is good >:D

Reply:
If you want to try and use an nvidia GPU as an opencl device with the visual mesh this will be needed